moped-editor/src/views/projects/projectView/ProjectActivityLogTableMaps.js

```diff
@@ -555,6 +555,24 @@ export const ProjectActivityLogTableMaps = {
       is_deleted: {
         label: "is deleted",
       },
+      interim_work_activity_id: {
+        label: "interim work activity ID",
+      },
+      implementation_workgroup: {
+        label: "implementation workgroup",
+      },
```

Comment:
this field is not actually implemented in the work activity form. I ultimately decided to use contractor to capture this. it's probably good to leave this here until we remove that field—but would you also rename the "Contractor" field to "Workgroup/Contractor"?

i will open an issue to remove this unused field from the db 😑